static/usage/select/typeahead/angular/example_component_ts.md

```diff
@@ -0,0 +1,111 @@
+```ts
+import { Component, ViewChild } from '@angular/core';
+import { IonModal } from '@ionic/angular';
+
+interface Fruit {
+  text: string;
+  value: string;
+}
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+})
+export class ExampleComponent {
```

Comment:
~ nit: Angular tends to omit `public` in their [style guide](https://angular.io/guide/styleguide) as definitions are public by default.

Reply:
fixed in https://github.com/ionic-team/ionic-docs/pull/2704/commits/d63f6a3cacafad503ecf852e2880b1dbed54d70b